src/user/user.spec.js

```diff
@@ -54,4 +54,24 @@ describe("user :", () => {
       await expect(xo.createUser(simpleUser)).rejects.toMatchSnapshot();
     });
   });
+
+  describe(".delete() :", () => {
+    it("deletes a user successfully with id", async () => {
+      const userId = await xo.call("user.create", simpleUser);
+      expect(await xo.call("user.delete", { id: userId })).toBeTrue();
```

Comment:
I can't find `.toBeTrue()` in the documentation… Do you have a link?

And IMHO, I prefer to always use `.toBe(true)`.

Reply:
https://www.npmjs.com/package/jest-extended#tobetrue
I will change that :+1:

Reply:
I see, but I don't find any advantages of `toBeTrue()` over `toBe(true)`, so maybe we should just the last one to avoid depending on `jest-extended`?

Reply:
OK. For the `jest-extended`, I will change that in another PR :+1:

Reply:
https://jestjs.io/docs/en/expect#tobetruthy

Reply:
This is different, it tests `expect(Boolean(value)).toBe(true)` and not `expect(value).toBe(true)`.

I prefer tests to be as precise as possible.

Reply:
Right, it's not precise than `toBe(true)` :+1: